packages/react-components/react-shared-contexts/src/ProviderContext/ProviderContext.ts

```diff
@@ -11,17 +11,18 @@ export type ProviderContextValue = {
 /**
  * @internal
  */
-// eslint-disable-next-line @fluentui/no-context-default-value
-const ProviderContext = React.createContext<ProviderContextValue>({
+const ProviderContext = React.createContext<ProviderContextValue | undefined>(undefined);
+
+const providerContextDefaultValue = {
```

Comment:
```suggestion
const providerContextDefaultValue: ProviderContextValue = {
```

This probably will remove need in `as 'ltr'` below